reference.md

```diff
@@ -612,7 +610,7 @@ Imagine a distance-based fare. The first 10 kilometers cost 3.30 CAD per kilomet
 
 ### wait_times.json
 
-This file defines wait times for the entire system via either zones in `zones.json` or S2 cells. To provide wait times to consumers, either this method or `wait_time` method can be used. `wait_times.json` allows lower server load on on demand system's servers at the cost of potentially lower precision. 
+This dynamic query provides wait time for specific location. To provide wait times to consumers, either this method or `wait_times.json` method can be used. `wait_time` allows more precise queries but requires a call on every interaction by users.
```

Comment:
To clarify - this PR does _not_ intend to change the (quasi) static nature of the `wait_times.json` file, correct? I'm not sure I understand the intention behind changing the description here

Reply:
Sorry, this was a typo. No changes to wait_times.json.

Reply:
This part has been reverted